modules/datablocksBidAdapter.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+```
+Module Name: Datablocks Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: support@datablocks.net
+```
+
+# Description
+
+Connects to Datablocks Version 5 Platform
+Banner Native and 
+
+
+# Test Parameters
+```
+    var adUnits = [
+      {
+        code: 'banner-div',
+        sizes: [[300, 250]],
+        mediaType:{
```

Comment:
Could you update the name here to `mediaTypes`?

Reply:
Also done in latest commit!